FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1453,4 +1453,81 @@ public function testCategoryNamesWithSlashesAreExportedCorrectly($articleConfigu
             $cat_urls
         );
     }
+
+    public function productsWithPseudoSalesProvider()
+    {
+        return [
+            'Pseudo sale value of 0 is not exported' => [
+                'pseudoSales' => 0,
+                'expectedSalesFrequency' => 1
```

Comment:
Names here should match the actual method parameter names. Either rename this here to `expected` or rename the method parameter to `$expectedSalesFrequency`.